examples/confetti/src/main/scala/com/example/confetti/Confetti.scala

```diff
@@ -1,6 +1,7 @@
 package com.example.confetti
 
 import indigo._
+import indigo.shared.events.MouseButton
```

Comment:
If you haven't already, could you please add an alias to this inside `package.scala`? Then you won't need this import.